Automattic/wp-calypso
# Subscribers Page: add sorting

Closes https://github.com/Automattic/wp-calypso/issues/78362

## Proposed Changes

This PR adds the sorting dropdown to the Subscribers Page:

<img width="1149" alt="Screenshot 2023-06-23 at 11 31 51 (1)" src="https://github.com/Automattic/wp-calypso/assets/3832570/da7a28df-b2e0-4fd1-bf20-fe2e37e25a16">

## Testing Instructions

1. Apply this PR and start the application.
2. Go to `http://calypso.localhost:3000/subscribers/{domain of your site}`.
3. Click on the sorting control and try the 2 possible values. The list should correctly order in each case.

## Pre-merge Checklist
